packages/sandbox/package.json

```diff
@@ -9,13 +9,12 @@
     "start": "next start"
   },
   "devDependencies": {
-    "babel-plugin-styled-components": "^2.0.6",
     "next": "^12.2.0"
   },
   "dependencies": {
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "styled-components": "*"
```

Comment:
This shouldn't be required since we're using yarn workspaces...

Reply:
`*` is only supported in Yarn Berry AFAIK. Would you like a PR upgrading to that or maybe `pnpm` (which I prefer)?